protokube/pkg/gossip/mesh/gossip.go

```diff
@@ -41,14 +41,16 @@ type MeshGossiper struct {
 
 func NewMeshGossiper(listen string, channelName string, nodeName string, password []byte, seeds gossip.SeedProvider) (*MeshGossiper, error) {
 
-	connLimit := 64
+	connLimit := 0 // 9 means no limit
```

Comment:
Is this a typo? 

**9** means no limit?

Reply:
Thank you - will fix!